src/daemon/demo.sh

```diff
@@ -6,7 +6,7 @@ unset "DAEMON_OPTS[${#DAEMON_OPTS[@]}-1]" # remove the last element of the array
 PREPARE_OSD=$OSD_PATH
 ACTIVATE_OSD=$OSD_PATH
 # the following ceph version can start with a numerical value where the new ones need a proper name
-if [[ "$CEPH_VERSION" == "jewel" || "$CEPH_VERSION" == "kraken" || "$CEPH_VERSION" == "luminous" ]]; then
+if [[ "$CEPH_VERSION" == "kraken" || "$CEPH_VERSION" == "luminous" ]]; then
```

Comment:
I think this should stay, just in case.

Reply:
As long as jewel is removed from master, it is no more possible to run that code inside a jewel container. So keeping dead code isn't a good idea. Let's remove what is deprecated. If some wants some jewel, let's use the stable branches then.

Reply:
Ack to resolve this part ?

Reply:
Yes!